doc/devel/contributing.rst

```diff
@@ -0,0 +1,422 @@
+.. _contributing:
+
+============
+Contributing
+============
+
+This project is a community effort, and everyone is welcome to
+contribute.
+
+The project is hosted on https://github.com/matplotlib/matplotlib
+
+Submitting a bug report
+=======================
+
+In case you experience issues using this package, do not hesitate to submit a
+ticket to the
+`Bug Tracker <https://github.com/matplotlib/matplotlib/issues>`_. You are
+also welcome to post feature requests or pull requests.
```

Comment:
Especially if we continue with "experience issues", then it's even more prudent to mention it.